portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,16 +32,9 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that handles comments data */
```

Comment:
Great job updating the comment. Method and class documentation in Java should end with a period. So you can update this to "Servlet that handles comments data.".

Reply:
Done